app/views/creatives/show.html.erb

```diff
@@ -3,7 +3,7 @@
   tabs: false,
   buttons: [
     layout_button(link: new_creative_path(clone: @creative.id), icon: "fad fa-copy", title: "Clone creative"),
-    layout_button(link: edit_creative_path(@creative), icon: "fad fa-pencil", title: "Edit creative"),
+    layout_button(link: edit_creative_path(@creative), icon: "fad fa-pencil", title: "Edit creative", admin: @creative.active?),
```

Comment:
The `admin` argument is somewhat confusing. I wonder if we could think of better semantics for it.

Reply:
I was thinking about that when I wrote this. I'll put it on my list.